src/shared/components/mutationMapper/MutationMapper.tsx

```diff
@@ -487,14 +494,12 @@ export default class MutationMapper<P extends IMutationMapperProps> extends Reac
     @action
     protected open3dPanel() {
         this.trackVisibility[TrackNames.PDB] = 'visible';
-        this.props.store.pdbChainDataStore.selectFirstChain();
     }
 
     @autobind
     @action
     protected close3dPanel() {
         this.trackVisibility[TrackNames.PDB] = 'hidden';
-        this.props.store.pdbChainDataStore.selectUid();
```

Comment:
We don't need this, since the `ProteinChainPanel` is not rendered anymore when the visibility is set to `hidden`.